CHANGELOG.md

```diff
@@ -1,146 +0,0 @@
-# Change Log
```

Comment:
Yes, we do! Ideally, it should be automatic (it was generated I think with [github-changelog-generator](https://github.com/skywinder/github-changelog-generator)).